### PR TITLE
Fix some grammar.

### PR DESCRIPTION
**Pull request by awesome-doge**

### CODE REVIEW

File: docs/func/literals_identifiers.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ FunC allows decimal and hexadecimal integer literals (leading zeros are allowed)
 For example, `0`, `123`, `-17`, `00987`, `0xef`, `0xEF`, `0x0`, `-0xfFAb`, `0x0001`, `-0`, `-0x0` are valid number literals.
 
 ## String literals
-Strings in FunC are quoted in double quotes `"` like `"this is a string"`. Special symbols like `\n` and multi-line stings are not supported. String are used only in asm functions definitions.
+Strings in FunC are quoted in double-quotes `"` like `"this is a string"`. Special symbols like `\n` and multi-line strings are not supported. String are used only in asm functions definitions.
```
Comment (akifoq) [inline docs/func/literals_identifiers.md]: I believe it's "double quotes" (without hyphen), for example you can find such a spelling in Wikipedia [article](https://en.wikipedia.org/wiki/Quotation_mark).

Comment (akifoq) [reply docs/func/literals_identifiers.md]: By the way, thanks for fixing my mistakes xD

File: docs/func/statements.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ int y = foo (1) (2) (3); ;; ok
 Lambda expressions are not supported yet.
 ### Methods calls
 #### Non-modifying methods
-If a function has at least one argument, it can be called as non-modifying method. For example, `store_uint` has type `(builder, int, int) -> builder` (the second argument is value to store, and the third is the bit length). `begin_cell` is a function that creates a new builder. The following codes are equivalent:
+If a function has at least one argument, it can be called a non-modifying method. For example, `store_uint` has type `(builder, int, int) -> builder` (the second argument is value to store, and the third is the bit length). `begin_cell` is a function that creates a new builder. The following codes are equivalent:
```
Comment (akifoq) [inline docs/func/statements.md]: I meant the function call expression (the function can act as a non-modifying method in such an expression), not the name of the function :)

File: docs/func/stdlib.md [hunk 3]
```diff
@@ -562,7 +562,7 @@ Decomposes `slice s` containing a valid `MsgAddress` into a `tuple t` with separ
 ```
 (int, int) parse_std_addr(slice s) asm "REWRITESTDADDR";
 ```
-Parses slice `s` containing a valid `MsgAddressInt` (usually a `msg_addr_std`), applies rewriting from the `anycast` (if present) to the same-length prefix of the address, and returns both the workchain and the 256-bit address as integers. If the address is not 256-bit, or if `s` is not a valid serialization of `MsgAddressInt`, throws a cell `deserialization` exception.
+Parses slice `s` containing a valid `MsgAddressInt` (usually a `msg_addr_std`), applies to rewrite from the `anycast` (if present) to the same-length prefix of the address, and returns both the workchain and the 256-bit address as integers. If the address is not 256-bit, or if `s` is not a valid serialization of `MsgAddressInt`, throws a cell `deserialization` exception.
```
Comment (akifoq) [inline docs/func/stdlib.md]: I believe "applies rewriting" is the correct spelling (i.e. it does the rewriting). It's borrowed from Dr. Nikolai Durov paper.

File: docs/func/types.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ Type of form `(A)` is considered by type-checker as the same type as `A`.
 Types of the form `[A, B, ...]` represent TVM tuples with concrete length and types of components, known in compile time. For example, `[int, cell]` is the type of TVM tuples, having length exactly 2, which first component is an integer, and the second is a cell. `[]` is the type of empty tuples (having the unique inhabitant – the empty tuple). Note that in contrast to unit type `()`, the value of `[]` occupy 1 stack entry.
 
 ## Polymorphism with type variables
-FunC has Miller-Rabin type system with support for polymorphic functions. For example, function
+FunC has Miller-Rabin-type system with support for polymorphic functions. For example, function
```
Comment (akifoq) [inline docs/func/types.md]: Miller and Rabin are the surnames of creators of the specific type system.

File: docs/howto/config-params.md [hunk 5]
```diff
@@ -206,18 +206,18 @@ After waiting for some time, we can inspect the incoming messages of our wallet
...
 
 We can inspect the proposed value for configuration parameter `#8` by asking the lite-client to expand cell `C{FDCD...}` using its hash `FDCD...` or a sufficiently long prefix of this hash to uniquely identify the cell in question:
 ```
-> dumpcell FDC
+> dump cell FDC
```
Comment (akifoq) [inline docs/howto/config-params.md]: `dumpcell` is a command of lite-client

File: docs/func/literals_identifiers.md (hunk 1 above)
Comment (awesome-doge) [reply docs/func/literals_identifiers.md]: fix

**Pull request merged**